chirpy/api.py

```diff
@@ -146,9 +146,15 @@ def __call__(self, **kwargs):
             uriparts.append(str(kwargs.pop(uripart, uripart)))
         uri = '/'.join(uriparts)
 
+        json_body = kwargs.pop('json', None)
```

Comment:
Minor nit, could you not clash with `json` since that's a builtin module name? `_json` would be fine.